### PR TITLE
PORTS NSV POOLS FOR REALSIES THIS TIME

### PR DESCRIPTION
**Pull request by maxion12345**
had to just delete the other branch shit was turbofucked my man

todo
- [ ] snip
snip
- [x] Throw people into pools
currently the pool acts like an invisible wall and you will be stopped
# Document the changes in your pull request
Port of https://github.com/BeeStation/BeeStation-Hornet/pull/3160

Adds pools from NSV13, replacing the boxing rings on each station. The holodeck's boxing ring is bigger and more sexy looking anyways. 
# Wiki Documentation
update photo

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  PowerfulBacon, KMC, Cdey, Maxim
rscadd: Added pool to boxstation, metastation, and the luxury shuttle (no more jpeg of a beach)
/:cl:

### CODE REVIEW

File: code/modules/pool/pool_items.dm [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
+
+/obj/item/twohanded/required/pool/Initialize()
+	. = ..()
+	//Pick a random color
+	color = pick(COLOR_YELLOW, COLOR_LIME, COLOR_RED, COLOR_BLUE_LIGHT, COLOR_CYAN, COLOR_MAGENTA)
+
+/obj/item/twohanded/required/pool/rubber_ring
+	name = "inflateable ring"
```
Comment (patpol4) [inline code/modules/pool/pool_items.dm]: inflateable? not inflatable?

File: code/modules/pool/pool_items.dm [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	desc = "An inflateable ring used for keeping people afloat. Throw at drowning people to save them."
```
Comment (patpol4) [inline code/modules/pool/pool_items.dm]: here too

**Review by patpol4 — CHANGES_REQUESTED** [review]
Double pipe on box here:
firefox_719bX3f27k

For meta:
firefox_nKx3GGclBE
Fix the pointless T junction, 
check for duplicates, 
move the line one to the left so its not so close to the pool
move the table so it's not right next to the pool, also maybe chairs? idk might change those later if it's bad ingame

Also move assistant spawns on both maps a bit so they dont spawn right next to the pools and instantly fall in there by accident.

**Comment by maxion12345** [comment]
> Double pipe on box here: firefox_719bX3f27k
> 
> For meta: firefox_nKx3GGclBE Fix the pointless T junction, check for duplicates, move the line one to the left so its not so close to the pool move the table so it's not right next to the pool, also maybe chairs? idk might change those later if it's bad ingame
> 
> Also move assistant spawns on both maps a bit so they dont spawn right next to the pools and instantly fall in there by accident.

done, I moved that uniform on the table over to where the cups used to be (you can get cups from the water machine idk why they had a table)

**Review by patpol4 — APPROVED** [review]
Map is fine

**Comment by patpol4** [comment]
previous PR https://github.com/yogstation13/Yogstation/pull/13114

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**